pkg/test/extensions/binary.go

```diff
@@ -34,6 +34,8 @@ type TestBinary struct {
 	// The binary path to extract from the image
 	binaryPath string
 
+	ext string
```

Comment:
`s/ext/extension/` please and add a comment when it should be specified

Reply:
thanks. update it. please help review it again.

Reply:
FYI. https://github.com/openshift/origin/compare/52ed9a1f8768b89d371f1170167670643947e6ba..86b12a6cc26dbc8f7e23706212dce7352828a68f